### ISSUE TITLE
Indexer's status is wrong since Indexer HTTP server has been removed

### ISSUE THREAD

**Issue by jsilvestre** [cozy/cozy-dev#79]
To reflect recent changes on indexer, cozy-dev should be changed in order to remove the check (that will fail).

**Comment by clochix** [comment]
cozy-dev uses the `/status` API of the proxy, so shouldn't this be fixed into the proxy?
See cozy/cozy-proxy#228

**Comment by jsilvestre** [comment]
Yes indeed, but I'd like to keep this issue open to make sure there is no reference at all to cozy-indexer in the code.
